src/sentry/testutils/pytest/sentry.py

```diff
@@ -50,7 +50,10 @@ def configure_split_db() -> None:
     settings.DATABASE_ROUTERS = ("sentry.db.router.SiloRouter",)
 
 
-DEFAULT_SILO_MODE_FOR_TEST_CASES = SiloMode.MONOLITH
+SENTRY_USE_MONOLITH_DBS = os.environ.get("SENTRY_USE_MONOLITH_DBS", "0") == "1"
+DEFAULT_SILO_MODE_FOR_TEST_CASES = (
+    SiloMode.REGION if not SENTRY_USE_MONOLITH_DBS else SiloMode.MONOLITH
+)
```

Comment:
I think I'd prefer to have `DEFAULT_SILO_MODE_FOR_TEST_CASES` remain a constant, and do

```python
    settings.SILO_MODE = (
        DEFAULT_SILO_MODE_FOR_TEST_CASES if not SENTRY_USE_MONOLITH_DBS else SiloMode.MONOLITH
    )
```
in `pytest/sentry.py`.